src/ctia/http/routes/crud.clj

```diff
@@ -229,45 +231,59 @@
                 :auth-identity identity
                 :identity-map identity-map
                 (GET "/histogram" []
-                     :return [{:key s/Str :value s/Int}]
+                     :return AggResult
```

Comment:
As the name is presented to users in SwaggerUI, I think you could use a longer name here:
`AggregationResult`. You might perhaps provide even more schema insight by having a specific
`HistogramResult` or `Histogram` schema here.
My guess is that each kind of aggregation share a common pattern but have a few useful distinctions
that could be shared via the schema.